packages/accordions/src/styled/accordion/StyledPanel.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+export interface IStyledPanel {
+  isBare?: boolean;
+  isCompact?: boolean;
+  isExpanded?: boolean;
+}
+
+const COMPONENT_ID = 'accordions.panel';
+
+const paddingStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
+  const { base } = props.theme.space;
+  let paddingTop = base * 2;
+  let paddingHorizontal = base * 5;
+  let paddingBottom = base * 8;
+
+  if (props.isCompact) {
+    paddingTop = base * 2;
+    paddingHorizontal = base * 3;
+    paddingBottom = base * 4;
+  }
+
+  if (props.isExpanded === false) {
+    paddingTop = 0;
+    paddingBottom = 0;
+  }
+
+  return css`
+    transition: padding 0.25s ease-in-out;
+    padding: ${paddingTop}px ${paddingHorizontal}px ${paddingBottom}px;
+  `;
+};
+
+export const StyledPanel = styled.section.attrs<IStyledPanel>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledPanel>`
+  ${paddingStyles};
+  border-bottom: ${props =>
+    `1px solid ${props.isBare ? 'transparent' : getColor('neutralHue', 300, props.theme)}`};
```

Comment:
Border `1px solid` should be getting pulled from `props.theme.borders.sm`.

Reply:
Thanks, updated 🎉 .